### PR TITLE
[CI] [GHA] Unskip `ov_subgraphs_dumper_tests` in Sanitizer workflow, fix dependency issue

### PR DESCRIPTION
**Pull request by akashchi**
### Tickets:
 - *134419*
 - *134423*

### CODE REVIEW

File: .github/workflows/linux_sanitizers.yml [hunk 1]
```diff
@@ -263,7 +263,7 @@ jobs:
       - name: Install dependencies (Linux)
         run: |
           $INSTALL_DIR/install_dependencies/install_openvino_dependencies.sh -c=core -c=dev -c=gpu -y
-          apt update && apt --assume-yes install clang lld
+          apt update && apt --assume-yes install clang lld libtbb2
```
Comment (akashchi) [inline .github/workflows/linux_sanitizers.yml:266]: W/o that the tests failed with `error while loading shared libraries: libtbb.so.2: cannot open shared object file: No such file or directory` e.g., in https://github.com/openvinotoolkit/openvino/actions/runs/9440578673/job/26002833869#step:11:18

Comment (ilya-lavrenov) [reply .github/workflows/linux_sanitizers.yml:266]: please, drop `ENABLE_SYSTEM_TBB=ON` in this file

Comment (akashchi) [reply .github/workflows/linux_sanitizers.yml]: Removed, will see how it goes: https://github.com/openvinotoolkit/openvino/actions/runs/9495998089

File: .github/workflows/linux_sanitizers.yml [hunk 2]
```diff
@@ -135,7 +135,7 @@ jobs:
             -DENABLE_NCC_STYLE=OFF \
             -DENABLE_TESTS=ON \
             -DENABLE_STRICT_DEPENDENCIES=OFF \
-            -DENABLE_SYSTEM_TBB=ON \
+            -DENABLE_SYSTEM_TBB=OFF \
```
Comment (ilya-lavrenov) [inline .github/workflows/linux_sanitizers.yml:138]: drop it at all

**Comment by akashchi** [comment]
It is green: https://github.com/openvinotoolkit/openvino/actions/runs/9513411009/job/26223468380?pr=24926.

Removed the PR trigger.

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**